hermes/cmd/hermes_configure.go

```diff
@@ -0,0 +1,585 @@
+	if err := hermes.UnmarshalResult(bufClientAResult.Bytes(), &clientAResult); err != nil {
+		return err
+	}
+
+	_ = session.Println(color.Green.Sprintf(
+		"Client '%s' created (%s -> %s)",
+		clientAResult.CreateClient.ClientID,
+		chainAID,
+		chainBID,
+	))
+
+	// create client B
+	var (
+		bufClientBResult = bytes.Buffer{}
+		clientBResult    = hermes.ClientResult{}
+	)
+	err = h.CreateClient(
+		cmd.Context(),
+		chainBID,
+		chainAID,
+		hermes.WithConfigFile(cfgPath),
+		hermes.WithStdOut(&bufClientBResult),
+	)
+	if err != nil {
+		return err
+	}
+	if err := hermes.UnmarshalResult(bufClientBResult.Bytes(), &clientBResult); err != nil {
+		return err
+	}
+
+	_ = session.Println(color.Green.Sprintf(
+		"Client %s' created (%s -> %s)",
+		clientBResult.CreateClient.ClientID,
+		chainBID,
+		chainAID,
+	))
+	session.StartSpinner("Creating connection")
+
+	// create connection
+	var (
+		bufConnection = bytes.Buffer{}
+		connection    = hermes.ConnectionResult{}
+	)
+	err = h.CreateConnection(
+		cmd.Context(),
+		chainAID,
+		clientAResult.CreateClient.ClientID,
+		clientBResult.CreateClient.ClientID,
+		hermes.WithConfigFile(cfgPath),
+		hermes.WithStdOut(&bufConnection),
+	)
+	if err != nil {
+		return err
+	}
+	if err := hermes.UnmarshalResult(bufConnection.Bytes(), &connection); err != nil {
+		return err
+	}
+
+	_ = session.Println(color.Green.Sprintf(
+		"Connection '%s (%s) <-> %s (%s)' created",
+		chainAID,
+		connection.ASide.ConnectionID,
+		chainBID,
+		connection.BSide.ConnectionID,
+	))
+	session.StartSpinner("Creating channel")
+
+	// create and query channel
+	var (
+		bufChannel = bytes.Buffer{}
+		channel    = hermes.ConnectionResult{}
+	)
+	err = h.CreateChannel(
+		cmd.Context(),
+		chainAID,
+		connection.ASide.ConnectionID,
+		chainAPortID,
+		chainBPortID,
+		hermes.WithConfigFile(cfgPath),
+		hermes.WithStdOut(&bufChannel),
+	)
+	if err != nil {
+		return err
+	}
+
+	if err := hermes.UnmarshalResult(bufChannel.Bytes(), &channel); err != nil {
+		return err
+	}
+
+	_ = session.Println(color.Green.Sprintf(
+		"Channel '%s (%s) <-> %s (%s)' created",
+		chainAID,
+		channel.ASide.ChannelID,
+		chainBID,
+		channel.BSide.ChannelID,
+	))
+
+	return nil
+}
+
+func verifyChainKeys(ctx context.Context, session *cliui.Session, h *hermes.Hermes, chainID, cfgPath string) error {
+	var (
+		bufKeysChain    = bytes.Buffer{}
+		keysChainResult = hermes.KeysListResult{}
+	)
+	if err := h.KeysList(
+		ctx,
+		chainID,
+		hermes.WithConfigFile(cfgPath),
+		hermes.WithStdOut(&bufKeysChain),
+	); err != nil {
+		return err
+	}
+	if err := hermes.UnmarshalResult(bufKeysChain.Bytes(), &keysChainResult); err != nil {
+		return err
+	}
+	if keysChainResult.Wallet.Account == "" {
+		var chainAMnemonic string
+		if err := session.Ask(cliquiz.NewQuestion(
+			fmt.Sprintf("Chain %s doesn't have a default Hermes key. Type your mnemonic to continue:", chainID),
+			&chainAMnemonic,
+			cliquiz.Required(),
+		)); err != nil {
+			return err
+		}
```

Comment:
An improvement, maybe for a new issue could be to generate one if the value is empty:

```go
fmt.Sprintf("Chain %s doesn't have a default Hermes key. Type your mnemonic to continue or leave it empty to generate one:", chainID),
```

Reply:
Makes sense 👍